oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -323,10 +323,10 @@ int mca_spml_ucx_add_procs(ompi_proc_t** procs, size_t nprocs)
 
     opal_progress_register(spml_ucx_default_progress);
 
-    mca_spml_ucx.remote_addrs_tbl = (char ***)calloc(mca_spml_ucx.ucp_workers, sizeof(char ***));
+    mca_spml_ucx.remote_addrs_tbl = (char ***)calloc(mca_spml_ucx.ucp_workers, sizeof(void*));
     for (w = 0; w < ucp_workers; w++) {
-        mca_spml_ucx.remote_addrs_tbl[w] = (char **)calloc(nprocs, sizeof(char **));
-        memset(mca_spml_ucx.remote_addrs_tbl[w], 0, nprocs * sizeof(char **));
+        mca_spml_ucx.remote_addrs_tbl[w] = (char **)calloc(nprocs, sizeof(void*));
+        memset(mca_spml_ucx.remote_addrs_tbl[w], 0, nprocs * sizeof(void*));
```

Comment:
`memset()` isn't needed here, since we use `calloc()` for memory allocation

Reply:
removed